editor/inserter/menu.js

```diff
@@ -245,12 +276,13 @@ class InserterMenu extends Component {
 
 	switchTab( tab ) {
 		this.setState( { tab: tab } );
+		this.setSearchFocus();
```

Comment:
this was to try and make it easier to use for people who are mostly keyboard based. There's no way to switch between tabs with the keyboard right now, so to make it easier to find the block you want, I put the focus into search so you could either start typing the name of a block or start tabbing through the list without any further mouse clicks. Happy to take it out, but it saved a _lot_ of mouse movement when testing search and tabbing!